### ISSUE TITLE
Can we merge  one PR with two approvals, but no yet third approval?

### ISSUE THREAD

**Issue by chunfuwen** [avocado-framework/avocado-vt#2747]
@sathnaga

We may meet one scenario that one PR has two approvals, but then third reviewer add some comments and those comments had been replied, so can we merge this PR without pending for third approval?

**Comment by sathnaga** [comment]
@chunfuwen yes, make sense, probabaly we can wait for a daya or two for third reviews gets back but it takes longer then, as long other reviewers can confirm those comments are addressed, we shall merge them without waiting.

**Comment by pevogam** [comment]
As far as I know 2 approvals are the Avocado VT standard for being allowed to merge. A more intricate question is one about a single review due to lack of "maintainer coverage" of a particular area of its rather large code base.

I raised some concerns about such cases [here](https://github.com/avocado-framework/avocado-vt/pull/2413#issuecomment-691023391) and [here](https://github.com/avocado-framework/avocado-vt/pull/2413#issuecomment-696227408). @sathnaga I want to hear your opinion and if you have any ideas how to handle such cases.
